packages/dashboards/addon/components/dashboard-header.hbs

```diff
@@ -0,0 +1,130 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="container-full m-b-4">
+  <div class="row m-t-10">
+    <div class="xs-col-1-1 sm-col-1-2 col-1-5 p-x-0">
+      {{#if (feature-flag "enableDirectory")}}
+        <LinkTo @route="directory.my-data" class="dashboard-header__breadcrumb-link">Directory</LinkTo>
+      {{else}}
+        <LinkTo @route="dashboards" class="dashboard-header__breadcrumb-link">Dashboards</LinkTo>
+      {{/if}}
+      <NaviIcon @icon="angle-right" />
+    </div>
+  </div>
+  <div class="row">
+    <div class="xs-col-10-10 sm-col-10-10 col-3-10 p-x-0">
+        <div class="row">
+          {{!-- Editable title --}}
+          <div class="dashboard-header__page-title">
+            {{#if @dashboard.canUserEdit}}
+              <EditableLabel
+                class="dashboard-header__edit-title"
+                @value={{@dashboard.title}}
+                @onChange={{route-action "updateTitle"}}
+              />
+            {{else}}
+              <span>{{@dashboard.title}}</span>
+            {{/if}}
+            <FavoriteItem
+              class="dashboard-header__fav-icon"
+              @user={{get-user}}
+              @item={{@dashboard}}
+              @click={{route-action "toggleFavorite" @dashboard}}
+            />
+          </div>
+        </div>
+    </div>
+    <div class="xs-col-10-10 sm-col-10-10 col-7-10 m-t-10 p-x-0">
+        <div class="row dashboard-header__actions">
```

Comment:
do we need this `row`/element?
```suggestion
        <div class="dashboard-header__actions">
```

Reply:
The media queries apply only when I have the row element here

Reply:
hmm, that shouldn't be the case. Maybe you just need `flex`?